pkg/apis/operator/v1alpha1/tektonpipeline_defaults_test.go

```diff
@@ -47,7 +47,7 @@ func Test_SetDefaults_PipelineProperties(t *testing.T) {
 		RequireGitSshSecretKnownHosts:            ptr.Bool(false),
 		EnableTektonOciBundles:                   ptr.Bool(false),
 		EnableCustomTasks:                        ptr.Bool(true),
-		EnableApiFields:                          "stable",
+		EnableApiFields:                          "beta",
```

Comment:
@piyush-garg can you update the test values from `github.com/tektoncd/pipeline/pkg/apis/config` => `config.DefaultEnableAPIFields`

Reply:
@jkandasa in the tests if you see we specifically use the values by specifying, so we know about the changes and can take action accordingly.

Reply:
@piyush-garg ok, I thought, in the future if there is a change in `default-api-field `, no need to touch this code base.